src/electron/index.ts

```diff
@@ -1,6 +1,6 @@
 import { BrowserWindow, app } from 'electron'
 import * as dotenv from 'dotenv'
-import { baseUrl } from './url/base-url'
+import { baseUrl } from './url/base-url/base-url'
```

Comment:
@Sansossio  Why is the path ends with the name of the file twice?

Reply:
Because base-url file is inside "base-url" folder, in this way we can have the definition and the test grouped